spec/objects/analytics_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "spec_helper"
```

Comment:
@osetinsky ideally, we'd be *generating* these files *from* JSON Schema:

https://github.com/interagent/heroics

But in this case, I don't think validating against a JSON Schema makes sense, since we're pulling arbitrary JSON from an endpoint that we control.

For `POST` or `PUT`-ing to endpoints, it would be nice to validate the incoming bodies against a schema.

>  maybe not for this PR

I agree!